gui/main.py

```diff
@@ -53,7 +53,7 @@ def make_app(self):
 class BaseStationHandler(tornado.web.RequestHandler):
     def get(self):
         # self.write("Hi There")
-        self.render("../gui/index.html", title="Title", items=[])
+        self.render("../gui/index2.html", title="Title", items=[])
```

Comment:
Index2 refers to the new html and index refers to our old one

Reply:
Please rename old index to `index_old.html` and this one to `index.html`.